### DISCUSSION TITLE
When would argo-repo as secret object be released

### DISCUSSION THREAD

**Discussion by soumentrivedi** [argoproj/argo-cd#6784] [Q&A]
With reference to the latest version of documentation, https://argoproj.github.io/argo-cd/operator-manual/declarative-setup/#repositories, can someone kindly share when can we expect this to be in the ArgoCD release? Its kind of an important feature that would be very useful for users with limited access onto argocd namespace.

Also is there any plan to create Argo Application in separate namespace and be discovered by ArgoCD?

**Comment by jgwest** [comment]
Hi @soumentrivedi , this will be in the Argo CD v2.1, which should be released within the next couple weeks :crossed_fingers:
